### PR TITLE
FAQ

### PR DESCRIPTION
**Pull request by lijo-harness**
Added FAQ

### CODE REVIEW

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -8072,4 +8072,5 @@ No, As per the current design it's not possible.
 #### How do I filter out Approvals for Pipeline Execution Time in Dashboards?
 You can get the Approval step duration from the **Deployments and Services V2** data source.
 
-
+#### When making changes to a pipeline, can the Github PR get created as a Github user?
+Yes, you could integrate oauth with git experience. More details about the same can be found in the [doc](https://developer.harness.io/docs/platform/git-experience/oauth-integration/)
```
Comment (roshnisarangadharan) [inline kb/continuous-delivery/continuous-delivery-faqs.md:8076]:
```suggestion

```

Comment (roshnisarangadharan) [reply kb/continuous-delivery/continuous-delivery-faqs.md:8076]: @lijo-harness Removing this FAQ. You're using an older version of CD FAQ page. We have revamped CD FAQs now, and details are documented [here](https://harness.atlassian.net/wiki/spaces/PD/pages/21745434679/New+CD+FAQ+paths?atlOrigin=eyJpIjoiMDI5N2Y0ZDE2NmZkNGViNDg5MmIzMjZmNzUwNDZiN2EiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ). Please add FAQs to respective sections going forward.

Comment (roshnisarangadharan) [reply kb/continuous-delivery/continuous-delivery-faqs.md:8076]: Also, please don't raise PR mixing FAQs in multiple modules.

**Review by roshnisarangadharan — DISMISSED** [review]
Removed the CD query from this. Please raise a new PR with just CD module FAQs in the respective sections as discussed.

**Comment by douglas-j-bothwell** [comment]
FYI @roshnisarangadharan @SushrutHarness @krishi0408 I did an editorial review of the CI FAQs that @lijo-harness added and fixed the resulting merge conflicts. A CD writer needs to resolve the merge conflict (looks like the topic was renamed).

**Pull request closed without merging**